README.md

```diff
@@ -104,17 +105,17 @@ Specify one or more irrigation zones here. The component calculates irrigation d
 - Bucket: You can manipulate this calculated value.
 - Lead Time: In seconds. Time needed to warm up your irrigation system, e.g. time to establish a connection or start a pump etc.
 
-You can Update all zones and Callculate all.
+You can update and callculate all automatic zones.
 
 **Per Zone Settings**
 You can change any value mentioned before. Additionaly there are some more options.
 
-- State: Wether Updating and Calculation of that zone should be enabled in mode 'Automatic' or 'Manual' or 'Disabled'
-- Module: Choose the calculation module (see below) shall be used for that zone to calculate irrigation duration
-- Mapping: Which sensor mapping (see below) shall be used for that zone
+- State: Wether updating and calculation of that zone should be enabled in mode 'Automatic' or 'Manual' or 'Disabled'.
```

Comment:
This is not what State is for.
If a zone is in automatic, it is indeed part of the auto update and auto calc. In other words, weather data is automatically collected and the duration is automatically calculated. In Manual mode, that doesn't happen and the user can enter a fixed duration (this is the replacement for the force mode in v1). In Disabled the zone doesn't have a duration and you can't set one either.

Reply:
done